clinicadl/train/train_utils.py

```diff
@@ -67,3 +67,40 @@ def get_user_dict(config_file: str, task: str) -> Dict[str, Any]:
         )
 
     return train_dict
+
+
+def get_model_list(architecture=None, input_size=(128, 128)):
+    """
+    Print the list of models available in ClinicaDL.
+    If architecture is given, it prints the details of the specified model.
+    """
+    from inspect import getmembers, isclass
+
+    from click import echo, secho
+
+    import clinicadl.utils.network as network_package
+
+    if not architecture:
+        echo("The list of currently available model is:")
+        model_list = getmembers(network_package, isclass)
+        for model in model_list:
+            echo(f" - {model[0]}")
+        echo(
+            "To show details of a specific model architecture please use the --architecture option"
+        )
+    else:
+        model_class = getattr(network_package, architecture)
+        args = list(
+            model_class.__init__.__code__.co_varnames[
+                : model_class.__init__.__code__.co_argcount
+            ]
+        )
+        args.remove("self")
+        kwargs = dict()
+        kwargs["input_size"] = (1, input_size[0], input_size[1])
```

Comment:
Actually this won't work for 3D architectures, and we might want to know the size of the FC layers for Conv5_FC3 in a 3D case.
It could be better not to be dependent of the dimenson, and just let the users set the input_size they want
```suggestion
        kwargs["input_size"] = input_size
```